src/server/pages.rs

```diff
@@ -64,22 +64,43 @@ pub fn load_file(file_name: &str) -> String {
     match file_name {
         "" | "index.html" => std::include_str!("../html/index.html").into(),
         "vue.js" => std::include_str!("../html/vue.js").into(),
+        "webrtc/index.html" => std::include_str!("../html/webrtc/index.html").into(),
```

Comment:
Be sure to remove all files that are incompatible with the MIT license, we can't include it at compile time.